nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -160,7 +160,10 @@ in {
     # If gnome3 is installed, build vim for gtk3 too.
     nixpkgs.config.vim.gui = "gtk3";
 
-    fonts.fonts = [ pkgs.dejavu_fonts pkgs.cantarell-fonts ];
+    fonts.fonts = [
+      pkgs.dejavu_fonts pkgs.cantarell-fonts
+      pkgs.source-sans-pro # Default monospace font in 3.32
```

Comment:
I think this needs to include `source-code-pro`-- which thankfully is, AFAICT, what's actually used for monospace (not somehow the sans variant? haha). Quick search on their gitlab turns up this:
https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/merge_requests/17/diffs#65231a5ac1ed586909f5137f1e9bdfe879aaa67b_160_159

(bit more info/background is on this issue: https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/issues/2 )

On my way to that page I noticed they are really highlighting "Source Sans" on their design policy page:
https://wiki.gnome.org/Engagement/BrandGuidelines

So I'd suggest pulling in both of them and possibly any other variants as well (?).
In a way I'm actually relieved, was worried some dark stuff was going to be in my debugging future re:"why does my terminal think a font with 'Sans' in the name is monospace?!" haha.

Reply:
Ah, nice catch, will fix :)